### PR TITLE
add functions to handle local storage

### PR DESCRIPTION
**Pull request by 1RV34**
`config(varname)` gets
`config(varname, value)` sets
`getConfig(varname, default)` gets or returns default if it doesn't exist
the setting ones are the same but are to store booleans (at some point localstorage will accept other types but for now localstorage only accept strings)

### CODE REVIEW

**Comment by 1RV34** [comment]
@DoctorMcKay what do you mean?

File: autoPlay.user.js [hunk 1]
```diff
@@ -16,11 +16,61 @@
...
+	var result = setting(name);
+
+	if (result === null &&
+		typeof defaultValue != "undefined")
+		return defaultValue;
+
+	return result;
+}
+
+var clickRate = getConfig("config/click-rate", 10);
```
Comment (xPaw) [inline autoPlay.user.js]: There's no need to make this a config variable, you can't go over 10 anyway.

**Comment by xPaw** [comment]
I see no reason to have multiple functions for this, just keep get/set.

**Comment by 1RV34** [comment]
removed clickrate from localstorage

@xPaw what exactly do you mean? you want me to put `config()` inside `getConfig()` and drop `config()` or something like that?

**Comment by DoctorMcKay** [comment]
Why are your functions anonymous?

**Comment by 1RV34** [comment]
@DoctorMcKay they are not anonymous, they're declared as variables inside the wrapping (anonymous..) function that acts as a scope

**Comment by DoctorMcKay** [comment]
They are anonymous functions assigned to variables.

**Comment by 1RV34** [comment]
@DoctorMcKay add me on steam so we can talk about it there instead of cluttering up this pull request ;) http://steamcommunity.com/id/1RV34

**Comment by DoctorMcKay** [comment]
Probably better to just join the [IRC channel](https://webchat.freenode.net/?channels=#steamdb).

**Comment by wchill** [comment]
Not needed anymore.

**Pull request closed without merging**